ui/src/runtime/components/Bake.vue

```diff
@@ -64,6 +60,19 @@ onUnmounted(() => {
   }
 });
 
+function render() {
+  const props = { };
+  if(descriptor.schema) { props.schema = descriptor.schema; }
+  if(descriptor.data) { props.data = data.value; }
+  if(descriptor.action) { props.onSubmit = onSubmit; }
+  if(is.props?.modelValue) {
+    props.modelValue = model.value;
+    props["onUpdate:modelValue"] = value => model.value = value;
+  }
+
+  return h(is, props);
+}
```

Comment:
now that we've done this, remove all redundant props and models from all components. that was added to avoid warning logs, so check removing props and models doesn't cause additional warnings